src/test/run-pass-fulldeps/proc-macro/attr-on-trait.rs

```diff
@@ -9,6 +9,7 @@
 // except according to those terms.
 
 // aux-build:attr-on-trait.rs
+// ignore-stage1
```

Comment:
why was this needed?

Reply:
This is the only test that fails on my machine (`x86_64-unknown-linux-gnu` with `./x.py test --stage 1`), and because it is in `run-pass-fulldeps`, running it only in `stage 2` doesn't seem a problem, because other tests in this directory are ignored on `stage 1`.